src/web/mjs/connectors/Alphapolis.mjs

```diff
@@ -43,11 +43,14 @@ export default class Alphapolis extends Connector {
-        `;
         const request = new Request(new URL(chapter.id, this.url), this.requestOptions);
-        return await Engine.Request.fetchUI(request, script);
+        const data = await this.fetchDOM(request, 'viewer-manga-horizontal');
+        try {
+            const pages = JSON.parse(data[0].getAttribute('v-bind:pages'));
+            return pages.filter(element => typeof element != 'object' && !element.match('white_page'));
```

Comment:
any exemple that explains the use of match ?

Reply:
The json gives us the pictures , but there are junk inside. Some elements are object we dont need, and there is another element that is a regular picture url but just a 'blank page" added at the end of each manga 

![image](https://user-images.githubusercontent.com/10671518/206757271-16a90947-a8e4-448b-8980-ae662fb11a39.png)